### PR TITLE
i18n(fr): update `reference/configuration.mdx`

### PR DESCRIPTION
**Pull request by HiDeoo**
#### What kind of changes does this PR include?

- Changes or translations of Starlight docs site content

#### Description

This PR updates the french version of the `reference/configuration.mdx` page with the changes of #742 and #1102.

### CODE REVIEW

File: docs/src/content/docs/fr/reference/configuration.mdx [hunk 1]
```diff
@@ -346,6 +345,72 @@ starlight({
 });
 ```
 
+### `expressiveCode`
+
+**Type :** `StarlightExpressiveCodeOptions | boolean`  
+**Par défaut :** `true`
+
+Starlight utilise [Expressive Code](https://github.com/expressive-code/expressive-code/tree/main/packages/astro-expressive-code) pour afficher les blocs de code et ajouter le support pour mettre en évidence des portions d'exemples de code, ajouter des noms de fichiers aux blocs de code, et plus encore.
+Consultez le [guide « Blocs de code »](/fr/guides/authoring-content/#blocs-de-code) pour apprendre à utiliser la syntaxe d'Expressive Code dans votre contenu Markdown et MDX.
```
Comment (HiDeoo) [inline docs/src/content/docs/fr/reference/configuration.mdx:354]: The link is currently broken in this PR, I'll update this comment with a reference to the PR fixing it.

Comment (HiDeoo) [reply docs/src/content/docs/fr/reference/configuration.mdx:354]: Updated the branch now that #1118 has been merged and the link is working properly in the preview.

**Review by delucis — APPROVED** [review]
ÇMSB ! 🌟 

Read through and nothing stuck out to me as a non-native French speaker.

**Review by dreyfus92 — APPROVED** [review]
LGTM ✅

**Pull request merged**